frontend/src/component/feature/FeatureStrategy/FeatureStrategyConstraints/FeatureStrategyConstraintAccordionList/FeatureStrategyConstraintAccordionList.tsx

```diff
@@ -109,7 +111,7 @@ export const FeatureStrategyConstraintAccordionList = forwardRef<
                                         }
                                     />
                                 </StyledHelpIconBox>
-                                <ConstraintList
+                                <NewConstraintAccordionList
```

Comment:
If the `newStrategyConfiguration` flag is true, we use the NewConstraintAccordionList with the autosave functionality